shiny/ui/_input_update.py

```diff
@@ -776,7 +776,7 @@ def update_slider(
 
     # Get any non-None value to see if the `data-type` may need to change
     val = value[0] if isinstance(value, tuple) else value
```

Comment:
Matching behavior lower in PR...

```suggestion
    val = value[0] if isinstance(value, (tuple, list)) else value
```